contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
 # Release Schedule
 
-- [Detailed schedule](#detailed-schedule)
-- [Determining if your change is in production](#determining-if-your-change-is-in-production)
-- [Cherry picks](#cherry-picks)
-  - [Cherry pick criteria](#cherry-pick-criteria)
-  - [Process for requesting a cherry pick](#process-for-requesting-a-cherry-pick)
-- [AMP Dev Channel](#amp-dev-channel)
-- [AMP Release Candidate (RC) Channel](#amp-release-candidate-rc-channel)
-- [Release cadence](#release-cadence)
-- [Release freezes](#release-freezes)
+- [Release Channels](#release-channels)
+  - [Weekly](#weekly)
+  - [Long-Term Stable (lts)](#long-term-stable-lts)
+- [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
+- [Release Cadence](#release-cadence)
+  - [Detailed schedule](#detailed-schedule)
+  - [AMP Dev Channel (Experimental)](#amp-dev-channel-experimental)
+  - [AMP Beta Channel (Beta)](#amp-beta-channel-beta)
+- [Release Freezes](#release-freezes)
 
 A new release of AMP is pushed to all AMP pages every week on Tuesday. **Once a change in AMP is merged into the master branch of the amphtml repository, it will typically take 1-2 weeks for the change be live for all users.**
 
-["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to canary testing to production). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+## Release Channels
 
-## Detailed schedule
+The AMP runtime and extensions are provided through a variety of different _release channels_. Each channel serves a purpose for developers and for the AMP HTML Project itself. See the [release cadence section](#release-cadence) for a more detailed breakdown of how and when code from the [`ampproject/amphtml`](https://github.com/ampproject/amphtml) repository makes it into release builds.
 
-We try to stick to this schedule as closely as possible, though complications may cause delays. You can track the latest status about any release in the ["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) and the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+See the [release cadence section](#release-cadence) for a more detailed breakdown of the release promotion cadence. To determine if a PR has been included in any of the following release channels, look for the GitHub labels _PR Use: In Canary_, _PR Use: In Production_, or _PR Use: In LTS_.
```

Comment:
Suggestion: link the label names to GH searches for those labels (can be done later, once the labels are regularly updated by the release tagger).

Reply:
I instead added a link to the "Determining if your change is in a release" section, which already includes an explanation of the labels along with links to each. LGTY?

Reply:
That works.